### PR TITLE
Adds a "shuffle/randomize cards" option to this library.

### PR DESCRIPTION
**Pull request by rezeau**
Signed-off-by: Joseph Rézeau <joseph@rezeau.org>

### CODE REVIEW

File: language/fr.json [hunk 1]
```diff
@@ -58,6 +58,10 @@
         {
           "label": "Mettre à l'échelle le texte afin de s'adapter au sein de la carte",
           "description": "Si cette option est décochée la taille de la carte sera adaptée au texte placé à l'intérieur"
+        },
+        {
+          "label": "Shuffle the Cards",
```
Comment (rezeau) [inline language/fr.json]: OK, Tim, I'll have another look at my PR and see if I can use the existing 'shuffle function' and I'll also test the  'Save content state' feature, but I might need help for that.
As for the language files I do agree they should only be changed once a PR has been merged (and the translation process has been agreed on).
QUESTION: Considering that the new changes might be important, should I create a brand new branch, called e.g. "Branch_randomize_cards" and then the current Branch_shuffle_cards could be deleted?

Comment (timothyylim) [inline language/fr.json]: I think we can continue on this branch :)

File: language/en.json [hunk 2]
```diff
@@ -0,0 +1,134 @@
+{
```
Comment (rezeau) [inline language/en.json]: Should I also remove @otacke 's changes to all lang files and German translation?

Comment (timothyylim) [inline language/en.json]: Yup, if possible.

**Comment by rezeau** [comment]
My commit a8e5420 makes most of the changes requested by @timothyylim. I don't know how to revert the German language file which had been committed by @otacke .
I've had a look at the 'Save content state' feature, but I'm really finding it difficult to implement. If @timothyylim can help me do it or do it himself, that would be fine.

**Comment by timothyylim** [comment]
Hi @rezeau, I'm sorry if I wasn't clear but the updates to the other language files should have remained the same but @otacke's new de.json should really be in another PR. 

On second thought, I think it's best that we leave the 'Save content state' feature to a later date, it's not too crucial for dialog cards at this point in time. 

I've pulled and merged your changes, but unfortunately, I did it in a way that GitHub doesn't understand so it may not show up on this thread. But you can look at the changes here:

https://github.com/h5p/h5p-dialogcards/commits/master

Thanks to you, the randomize feature should be available for everyone else in the next release 😺

**Comment by otacke** [comment]
Fear not! ;-) There's a PR for the German translation at https://github.com/h5p/h5p-dialogcards/pull/16 and I also tested Transifex for h5p-dialogcards.

**Pull request closed without merging**